data/json/items/tools.json

```diff
@@ -23,7 +23,29 @@
     "ammo": "NULL",
     "revert_to": "null"
   },
-
+  {
+      "id":"pet_carrier",
+      "type":"TOOL",
+      "name":"pet carrier",
+      "color":"light_gray",
+      "symbol":"#",
+      "description":"A plastic container meant to hold pets for transport.",
+      "price":1000,
+      "weight":1000,
+      "volume":20,
+      "to_hit":0,
+      "bashing":1,
+      "cutting":1,
+      "max_charges":0,
+      "initial_charges":0,
+      "turns_per_charge":0,
+      "charges_per_use":0,
+      "ammo":"NULL",
+      "revert_to":"null",
+      "use_action":"CAPTURE_MONSTER_ACT",
+      "flags":"NO_BREAK_ON_FAIL",
```

Comment:
That the container is not liable to break if the check for entrapping a monster fails. 
Are item flags meant to be declared globally? 
Regardless, I'll add at the very least a comment explaining it's meaning in the item entry.
It was part of the pokemon mod, so it can be removed at the moment.